visRayTracer.mac

```diff
@@ -0,0 +1,32 @@
+# Visualization and setup macro 
+# This file is run by default if you just type WCSim and nothing else.
+
+/run/verbose 0
+/tracking/verbose 0
+/hits/verbose 0
+
+##select RayTracer Visualization
+/WCSim/WCVisChoice RayTracer
+
+## select the geometry
+# Default config if you do nothing is currently SuperK
+#/WCSim/WCgeom SuperK
+
+/WCSim/Construct # This must be uncommented to use RayTracer
+
+## Errors may arise if using RayTracer and attempting to view the detector from a distance further than 40 meters
+## If this occurs, attempt changing the y-direction
+
+/vis/open RayTracer
+/vis/viewer/set/upVector 0 0 1
+
+/vis/rayTracer/eyePosition -15 0 -15
```

Comment:
Put a brief comment beside these eyePosition lines saying inside the ID looking toward the top corner, etc